student-work/assignment-8/Jason-Sandoval/helloworld.sh

```
@@ -0,0 +1,4 @@
+echo "hello world!"
```

Shellcheck (0.11.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)